frameworks/Crystal/kemal/shard.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
     version: "0.13.3"
   kemal:
     github: "sdogruyol/kemal"
-    version: "0.18.3"
+    branch: master
```

Comment:
The code would work forever as-is, because the version is locked down to a git sha in `shards.lock`. However as `0.19.0` has been released, we might as well use that version instead.